deploy/examples/object-openshift.yaml

```diff
@@ -103,6 +103,14 @@ spec:
       disabled: false
     readinessProbe:
       disabled: false
+  # To enable LDAP configurations for RGW
```

Comment:
I'd suggest let's not update object-openshfit.yaml for the ldap feature. We don't have to replicate all the settings in all the examples.